eos-updater-flatpak-installer/main.c

```diff
@@ -114,6 +121,18 @@ fail (int          exit_status,
   /* Include the usage. */
   g_printerr ("%s: %s\n", g_get_prgname (), formatted_message);
 
+  /* Report a metric. */
+#ifdef HAS_EOSMETRICS_0
```

Comment:
Doesn't eos-updater normally use libeufi rather than the eufi command, meaning a failure would not cross this code path?

Reply:
Correct, the `eos-updater` binary uses libeufi (well, actually libeuu) during its fetch stage to download flatpaks. However, any errors from that are propagated back to the D-Bus interface, and hence are all reported through the code path in `object.c` which this MR modifies.

The `eos-updater-flatpak-installer` binary is run as a service on first boot after an upgrade and actually installs/deploys/uninstalls/updates the flatpaks. Its errors need to be reported, but need to be reported separately (as they’re non-fatal; errors from `eos-updater` will halt the update).

Reply:
That makes sense. I forgot about or didn't know about that eufi service.